tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -829,41 +832,56 @@ montecarlo_main_loop(storage_model_t * storage, int64_t virtual_packet_flag, int
 #ifdef WITHOPENMP
   omp_set_dynamic(0);
   omp_set_num_threads(nthreads);
-#pragma omp parallel
+
+#pragma omp parallel firstprivate(finished_packets)
```

Comment:
i made finished_packets ( my variable to count ) firstprivate so it won't be uninitialized in the threads. I made it private so it won't impact performance.